src/AppStream.DurablePatterns/Steps/ConfigurationValidator/StepValidator.cs

```diff
@@ -14,15 +14,18 @@ public void Validate(Step stepConfiguration, Step? previousStepConfiguration)
                 return;
             }
 
-            var previousStepResultType = previousStepConfiguration.PatternActivityResultType;
-            var currentStepInputType = stepConfiguration.PatternActivityInputType;
+            var previousStepResultType = Type.GetType(previousStepConfiguration.PatternActivityResultTypeAssemblyQualifiedName)!;
```

Comment:
Why is it better than providing type itself? The serialization is the problem? Size of it?

Reply:
That's stupid, but well, what can you do